tests/api/conftest.py

```diff
@@ -61,3 +63,58 @@ def create_task_1(client, create_project_1):
 def create_task_2(client, create_project_1):
     rv = client.post('/tasks', data=json.dumps({'name': 'task_2', 'project_id': 1}))
     assert rv.status_code == 201
+
+
+@pytest.fixture
+def project_in_db(postgres_meta, s3_artifact): # noqa
+    proj = Project(name='test_project')
+    proj = postgres_meta.create_project(proj)
+    yield proj
+
+
+@pytest.fixture
+def task_in_db(postgres_meta, project_in_db, s3_artifact): # noqa
```

Comment:
also why noqa? if you use it, better specify errors you want to be ignored explicitly